src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -168,7 +168,7 @@ All parameters are case-insensitive.
     This can be any endpoint registered on the local machine including the default PowerShell
     remoting endpoints or a custom endpoint having specific user role capabilities.
 
-    Example: pwsh -configurationmame AdminRoles
+    Example: pwsh -configurationname AdminRoles
```

Comment:
Perhaps using `-ConfigurationName` would be better. The `nn` looks like a typo. -`ConfigurationName` is used elsewhere and there are plenty of examples of the parameters being case insensitive.